configs/defaults.yaml

```diff
@@ -0,0 +1,35 @@
+# Example configuration, copy file as "local.yaml" and edit as needed.
+
+# PostgreSQL
+# database:
+#   host: localhost
+#   port: 5432
+#   name:
+#   user:
+#   password:
+#   max_open_conn: 25
+#   max_idle_conn: 20
+#   max_lifetime: 5m
+#   max_idletime: 10m
+#   # 1-none, 2-error, 3-warn, 4-info, 5-debug, 6-trace
+#   loglevel: 1
+
+# logging:
+#   # Overall log level (-1 trace, 0 debug, 1 info ... 5 panic)
+#   level: 1
+#   # Write logs to standard output in human-readable form
+#   stdout: false
+
+# AWS CloudWatch logger
+# cloudwatch:
+#   enabled: false
+#   group: group
+#   stream: stream
+
+# When panic error is not rescued, exit the application instead of returning 500.
+
+# Feature flags:
+# development or production
+# feature_flags:
+#   environment: development
+#   exit_on_panic: false
```

Comment:
This needs to be moved to line 30.

Reply:
Why? I've put that to feature_flags in config as well

Reply:
Oh okay than move the comment to the appropriate place? I mean comments for flags are off.